[36m[2022-10-12 10:48:58 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:48:58 DEBUG] [cli:113][39m Running command 'run'
[36m[2022-10-12 10:48:58 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
[36m[2022-10-12 10:49:01 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at 'C:\Users\senur\.chaostoolkit\settings.yaml'.
[36m[2022-10-12 10:49:01 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:49:01 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:49:01 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:49:01 DEBUG] [caching:35][39m Cached 2 activities
[32m[2022-10-12 10:49:01 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2022-10-12 10:49:01 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:49:01 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:49:01 DEBUG] [secret:104][39m Done loading secrets
[32m[2022-10-12 10:49:03 INFO] [experiment:109][39m Experiment looks valid
[36m[2022-10-12 10:49:03 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:49:03 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:49:03 DEBUG] [caching:35][39m Cached 2 activities
[36m[2022-10-12 10:49:03 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:49:03 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:49:03 DEBUG] [secret:104][39m Done loading secrets
[36m[2022-10-12 10:49:03 DEBUG] [configuration:160][39m Loading dynamic configuration...
[36m[2022-10-12 10:49:03 DEBUG] [exit:90][39m System 'Windows-10-10.0.22000-SP0' does not expose SIGUSR signals
[32m[2022-10-12 10:49:03 INFO] [run:320][39m Running experiment: Lets kill pods randomly
[36m[2022-10-12 10:49:03 DEBUG] [__init__:52][39m Initializing controls
[32m[2022-10-12 10:49:03 INFO] [run:344][39m Steady-state strategy: default
[32m[2022-10-12 10:49:03 INFO] [run:348][39m Rollbacks strategy: default
[36m[2022-10-12 10:49:03 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:49:03 DEBUG] [run:506][39m Running steady-state hypothesis before the method
[32m[2022-10-12 10:49:03 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:49:03 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:49:03 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:49:03 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:49:03 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:49:03 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:49:03 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:49:11 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:49:11 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:49:11 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:49:11 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:49:11 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:49:11 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:49:11 INFO] [run:607][39m Playing your experiment's method now...
[36m[2022-10-12 10:49:11 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:49:11 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:49:11 INFO] [activity:204][39m Action: terminate-pod
[36m[2022-10-12 10:49:11 DEBUG] [python:33][39m Activity 'terminate-pod' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\pod\actions.py'
[36m[2022-10-12 10:49:11 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:49:42 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:49:42 DEBUG] [cli:113][39m Running command 'run'
[36m[2022-10-12 10:49:42 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
[36m[2022-10-12 10:49:45 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at 'C:\Users\senur\.chaostoolkit\settings.yaml'.
[36m[2022-10-12 10:49:45 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:49:45 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:49:45 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:49:45 DEBUG] [caching:35][39m Cached 2 activities
[32m[2022-10-12 10:49:45 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2022-10-12 10:49:45 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:49:45 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:49:45 DEBUG] [secret:104][39m Done loading secrets
[32m[2022-10-12 10:49:46 INFO] [experiment:109][39m Experiment looks valid
[36m[2022-10-12 10:49:46 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:49:46 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:49:46 DEBUG] [caching:35][39m Cached 2 activities
[36m[2022-10-12 10:49:46 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:49:46 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:49:46 DEBUG] [secret:104][39m Done loading secrets
[36m[2022-10-12 10:49:46 DEBUG] [configuration:160][39m Loading dynamic configuration...
[36m[2022-10-12 10:49:46 DEBUG] [exit:90][39m System 'Windows-10-10.0.22000-SP0' does not expose SIGUSR signals
[32m[2022-10-12 10:49:46 INFO] [run:320][39m Running experiment: Lets kill pods randomly
[36m[2022-10-12 10:49:46 DEBUG] [__init__:52][39m Initializing controls
[32m[2022-10-12 10:49:46 INFO] [run:344][39m Steady-state strategy: default
[32m[2022-10-12 10:49:46 INFO] [run:348][39m Rollbacks strategy: default
[36m[2022-10-12 10:49:46 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:49:46 DEBUG] [run:506][39m Running steady-state hypothesis before the method
[32m[2022-10-12 10:49:46 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:49:46 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:49:46 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:49:46 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:49:46 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:49:46 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:49:46 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:49:50 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:49:50 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:49:50 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:49:50 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:49:50 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:49:50 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:49:50 INFO] [run:607][39m Playing your experiment's method now...
[36m[2022-10-12 10:49:50 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:49:50 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:49:50 INFO] [activity:204][39m Action: terminate-pod
[36m[2022-10-12 10:49:50 DEBUG] [python:33][39m Activity 'terminate-pod' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\pod\actions.py'
[36m[2022-10-12 10:49:50 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:49:52 DEBUG] [actions:238][39m Found 12 pods in ns 'default'
[36m[2022-10-12 10:49:52 DEBUG] [activity:221][39m   => succeeded with '['redis-cart-78746d49dc-xx2fs']'
[32m[2022-10-12 10:49:52 INFO] [activity:239][39m Pausing after activity for 10s...
[36m[2022-10-12 10:50:02 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:50:02 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:50:02 DEBUG] [run:536][39m Running steady-state hypothesis after the method
[32m[2022-10-12 10:50:02 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:50:02 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:50:02 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:50:02 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:50:02 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:50:02 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:50:02 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:50:07 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:50:07 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:50:07 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:50:07 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:50:07 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:50:07 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:50:07 INFO] [run:893][39m Let's rollback...
[36m[2022-10-12 10:50:07 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:50:07 INFO] [rollback:27][39m No declared rollbacks, let's move on.
[36m[2022-10-12 10:50:07 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:50:07 INFO] [run:458][39m Experiment ended with status: completed
[36m[2022-10-12 10:50:07 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:50:07 DEBUG] [__init__:91][39m Cleaning up controls
[36m[2022-10-12 10:50:07 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:51:34 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:51:34 DEBUG] [cli:113][39m Running command 'run'
[36m[2022-10-12 10:51:34 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
[36m[2022-10-12 10:51:36 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at 'C:\Users\senur\.chaostoolkit\settings.yaml'.
[36m[2022-10-12 10:51:36 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:51:36 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:51:36 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:51:36 DEBUG] [caching:35][39m Cached 2 activities
[32m[2022-10-12 10:51:36 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2022-10-12 10:51:36 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:51:36 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:51:36 DEBUG] [secret:104][39m Done loading secrets
[32m[2022-10-12 10:51:37 INFO] [experiment:109][39m Experiment looks valid
[36m[2022-10-12 10:51:37 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:51:37 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:51:37 DEBUG] [caching:35][39m Cached 2 activities
[36m[2022-10-12 10:51:37 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:51:37 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:51:37 DEBUG] [secret:104][39m Done loading secrets
[36m[2022-10-12 10:51:37 DEBUG] [configuration:160][39m Loading dynamic configuration...
[36m[2022-10-12 10:51:38 DEBUG] [exit:90][39m System 'Windows-10-10.0.22000-SP0' does not expose SIGUSR signals
[32m[2022-10-12 10:51:38 INFO] [run:320][39m Running experiment: Lets kill pods randomly
[36m[2022-10-12 10:51:38 DEBUG] [__init__:52][39m Initializing controls
[32m[2022-10-12 10:51:38 INFO] [run:344][39m Steady-state strategy: default
[32m[2022-10-12 10:51:38 INFO] [run:348][39m Rollbacks strategy: default
[36m[2022-10-12 10:51:38 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:51:38 DEBUG] [run:506][39m Running steady-state hypothesis before the method
[32m[2022-10-12 10:51:38 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:51:38 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:51:38 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:51:38 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:51:38 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:51:38 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:51:38 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:51:40 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:51:40 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:51:40 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:51:40 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:51:40 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:51:40 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:51:40 INFO] [run:607][39m Playing your experiment's method now...
[36m[2022-10-12 10:51:40 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:51:40 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:51:40 INFO] [activity:204][39m Action: terminate-pod
[36m[2022-10-12 10:51:40 DEBUG] [python:33][39m Activity 'terminate-pod' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\pod\actions.py'
[36m[2022-10-12 10:51:40 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:51:43 DEBUG] [actions:238][39m Found 12 pods in ns 'default'
[36m[2022-10-12 10:51:44 DEBUG] [activity:221][39m   => succeeded with '['redis-cart-78746d49dc-gx659']'
[32m[2022-10-12 10:51:44 INFO] [activity:239][39m Pausing after activity for 10s...
[36m[2022-10-12 10:51:54 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:51:54 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:51:54 DEBUG] [run:536][39m Running steady-state hypothesis after the method
[32m[2022-10-12 10:51:54 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:51:54 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:51:54 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:51:54 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:51:54 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:51:54 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:51:54 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:51:56 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:51:56 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:51:56 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:51:56 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:51:56 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:51:56 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:51:56 INFO] [run:893][39m Let's rollback...
[36m[2022-10-12 10:51:56 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:51:56 INFO] [rollback:27][39m No declared rollbacks, let's move on.
[36m[2022-10-12 10:51:56 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:51:56 INFO] [run:458][39m Experiment ended with status: completed
[36m[2022-10-12 10:51:56 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:51:56 DEBUG] [__init__:91][39m Cleaning up controls
[36m[2022-10-12 10:51:56 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:52:31 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:52:31 DEBUG] [cli:113][39m Running command 'run'
[36m[2022-10-12 10:52:31 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
[36m[2022-10-12 10:52:32 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at 'C:\Users\senur\.chaostoolkit\settings.yaml'.
[36m[2022-10-12 10:52:32 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:52:32 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:52:32 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:52:32 DEBUG] [caching:35][39m Cached 2 activities
[32m[2022-10-12 10:52:32 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2022-10-12 10:52:32 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:52:32 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:52:32 DEBUG] [secret:104][39m Done loading secrets
[32m[2022-10-12 10:52:33 INFO] [experiment:109][39m Experiment looks valid
[36m[2022-10-12 10:52:33 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:52:33 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:52:33 DEBUG] [caching:35][39m Cached 2 activities
[36m[2022-10-12 10:52:33 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:52:33 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:52:33 DEBUG] [secret:104][39m Done loading secrets
[36m[2022-10-12 10:52:33 DEBUG] [configuration:160][39m Loading dynamic configuration...
[36m[2022-10-12 10:52:33 DEBUG] [exit:90][39m System 'Windows-10-10.0.22000-SP0' does not expose SIGUSR signals
[32m[2022-10-12 10:52:33 INFO] [run:320][39m Running experiment: Lets kill pods randomly
[36m[2022-10-12 10:52:33 DEBUG] [__init__:52][39m Initializing controls
[32m[2022-10-12 10:52:33 INFO] [run:344][39m Steady-state strategy: default
[32m[2022-10-12 10:52:33 INFO] [run:348][39m Rollbacks strategy: default
[36m[2022-10-12 10:52:33 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:52:33 DEBUG] [run:506][39m Running steady-state hypothesis before the method
[32m[2022-10-12 10:52:33 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:52:33 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:52:33 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:52:33 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:52:33 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:52:33 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:52:33 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:52:36 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:52:36 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:52:36 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:52:36 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:52:36 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:52:36 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:52:36 INFO] [run:607][39m Playing your experiment's method now...
[36m[2022-10-12 10:52:36 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:52:36 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:52:36 INFO] [activity:204][39m Action: terminate-pod
[36m[2022-10-12 10:52:36 DEBUG] [python:33][39m Activity 'terminate-pod' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\pod\actions.py'
[36m[2022-10-12 10:52:36 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:52:38 DEBUG] [actions:238][39m Found 12 pods in ns 'default'
[36m[2022-10-12 10:52:39 DEBUG] [activity:221][39m   => succeeded with '['checkoutservice-5d45565464-chqxx']'
[32m[2022-10-12 10:52:39 INFO] [activity:239][39m Pausing after activity for 10s...
[36m[2022-10-12 10:52:49 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:52:49 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:52:49 DEBUG] [run:536][39m Running steady-state hypothesis after the method
[32m[2022-10-12 10:52:49 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:52:49 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:52:49 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:52:49 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:52:49 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:52:49 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:52:49 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:52:51 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:52:51 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:52:51 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:52:51 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:52:51 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:52:52 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:52:52 INFO] [run:893][39m Let's rollback...
[36m[2022-10-12 10:52:52 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:52:52 INFO] [rollback:27][39m No declared rollbacks, let's move on.
[36m[2022-10-12 10:52:52 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:52:52 INFO] [run:458][39m Experiment ended with status: completed
[36m[2022-10-12 10:52:52 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:52:52 DEBUG] [__init__:91][39m Cleaning up controls
[36m[2022-10-12 10:52:52 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:56:21 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:56:21 DEBUG] [cli:113][39m Running command 'run'
[36m[2022-10-12 10:56:21 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
[36m[2022-10-12 10:56:24 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at 'C:\Users\senur\.chaostoolkit\settings.yaml'.
[36m[2022-10-12 10:56:24 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:56:24 DEBUG] [__init__:399][39m No controls to apply on 'loader'
[36m[2022-10-12 10:56:24 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:56:24 DEBUG] [caching:35][39m Cached 2 activities
[32m[2022-10-12 10:56:24 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2022-10-12 10:56:24 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:56:24 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:56:24 DEBUG] [secret:104][39m Done loading secrets
[32m[2022-10-12 10:56:26 INFO] [experiment:109][39m Experiment looks valid
[36m[2022-10-12 10:56:26 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:56:26 DEBUG] [caching:24][39m Building activity cache...
[36m[2022-10-12 10:56:26 DEBUG] [caching:35][39m Cached 2 activities
[36m[2022-10-12 10:56:26 DEBUG] [configuration:63][39m Loading configuration...
[36m[2022-10-12 10:56:26 DEBUG] [secret:78][39m Loading secrets...
[36m[2022-10-12 10:56:26 DEBUG] [secret:104][39m Done loading secrets
[36m[2022-10-12 10:56:26 DEBUG] [configuration:160][39m Loading dynamic configuration...
[36m[2022-10-12 10:56:26 DEBUG] [exit:90][39m System 'Windows-10-10.0.22000-SP0' does not expose SIGUSR signals
[32m[2022-10-12 10:56:26 INFO] [run:320][39m Running experiment: Lets kill pods randomly
[36m[2022-10-12 10:56:26 DEBUG] [__init__:52][39m Initializing controls
[32m[2022-10-12 10:56:26 INFO] [run:344][39m Steady-state strategy: default
[32m[2022-10-12 10:56:26 INFO] [run:348][39m Rollbacks strategy: default
[36m[2022-10-12 10:56:26 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:56:26 DEBUG] [run:506][39m Running steady-state hypothesis before the method
[32m[2022-10-12 10:56:26 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:56:26 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:56:26 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:56:26 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:56:26 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:56:26 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:56:26 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:56:28 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:56:28 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:56:28 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:56:28 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:56:28 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:56:28 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:56:28 INFO] [run:607][39m Playing your experiment's method now...
[36m[2022-10-12 10:56:28 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:56:28 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:56:28 INFO] [activity:204][39m Action: terminate-pod
[36m[2022-10-12 10:56:28 DEBUG] [python:33][39m Activity 'terminate-pod' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\pod\actions.py'
[36m[2022-10-12 10:56:28 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:56:31 DEBUG] [actions:238][39m Found 12 pods in ns 'default'
[36m[2022-10-12 10:56:32 DEBUG] [activity:221][39m   => succeeded with '['shippingservice-55bd6c45bb-x42xs']'
[32m[2022-10-12 10:56:32 INFO] [activity:239][39m Pausing after activity for 10s...
[36m[2022-10-12 10:56:42 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:56:42 DEBUG] [__init__:399][39m No controls to apply on 'method'
[36m[2022-10-12 10:56:42 DEBUG] [run:536][39m Running steady-state hypothesis after the method
[32m[2022-10-12 10:56:42 INFO] [hypothesis:182][39m Steady state hypothesis: The app is healthy
[36m[2022-10-12 10:56:42 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[36m[2022-10-12 10:56:42 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[32m[2022-10-12 10:56:42 INFO] [activity:204][39m Probe: all-apps-are-healthy
[36m[2022-10-12 10:56:42 DEBUG] [python:33][39m Activity 'all-apps-are-healthy' loaded from 'C:\Users\senur\AppData\Local\Programs\Python\Python310\lib\site-packages\chaosk8s\probes.py'
[33m[2022-10-12 10:56:42 WARNING] [__init__:165][39m all_microservices_healthy function is DEPRECATED and will be removed in the next         releases, please use all_pods_healthy instead
[36m[2022-10-12 10:56:42 DEBUG] [__init__:73][39m Using Kubernetes context 'default' from config 'C:\Users\senur/.kube/config'
[36m[2022-10-12 10:56:45 DEBUG] [probes:296][39m Found 0 failed and 0 not ready pods
[36m[2022-10-12 10:56:45 DEBUG] [activity:221][39m   => succeeded with 'True'
[36m[2022-10-12 10:56:45 DEBUG] [__init__:399][39m No controls to apply on 'activity'
[36m[2022-10-12 10:56:45 DEBUG] [hypothesis:221][39m allowed tolerance is True
[32m[2022-10-12 10:56:45 INFO] [hypothesis:231][39m Steady state hypothesis is met!
[36m[2022-10-12 10:56:45 DEBUG] [__init__:399][39m No controls to apply on 'hypothesis'
[32m[2022-10-12 10:56:45 INFO] [run:893][39m Let's rollback...
[36m[2022-10-12 10:56:45 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:56:45 INFO] [rollback:27][39m No declared rollbacks, let's move on.
[36m[2022-10-12 10:56:45 DEBUG] [__init__:399][39m No controls to apply on 'rollback'
[32m[2022-10-12 10:56:45 INFO] [run:458][39m Experiment ended with status: completed
[36m[2022-10-12 10:56:45 DEBUG] [__init__:399][39m No controls to apply on 'experiment'
[36m[2022-10-12 10:56:45 DEBUG] [__init__:91][39m Cleaning up controls
[36m[2022-10-12 10:56:45 DEBUG] [caching:42][39m Clearing activities cache
[36m[2022-10-12 10:57:46 DEBUG] [cli:112][39m ###############################################################################
[36m[2022-10-12 10:57:46 DEBUG] [cli:113][39m Running command 'report'
[36m[2022-10-12 10:57:46 DEBUG] [cli:117][39m Using settings file 'C:\Users\senur\.chaostoolkit\settings.yaml'
